src/views/Weather-Components/Constant.js

```diff
@@ -12,3 +12,18 @@ export const Cities = [
   { title: "Wazirabad", value: "wazirabad" },
   { title: "Attock", value: "attock" },
 ];
+
+export const popularCities = [
+  { city: "Lahore", province: "Punjab" },
+  { city: "Multan", province: "Punjab" },
+  { city: "Kirachi", province: "Sindh" },
+  { city: "Quetta", province: "Balochistan" },
+];
+
+export const Buttons = [

```

Comment:
This name `Button` doesn't imply the correct intent. It can be more like `DETAIL_TABLS` or something.